cdk/lib/elastic-search-monitor.ts

```diff
@@ -79,6 +85,10 @@ export class ElasticSearchMonitor extends GuStack {
         ),
       ],
     });
+
+    const cfnLambda = scheduledLambda.node.defaultChild as CfnFunction;
```

Comment:
I'm reluctant to add this to GuCDK now as I think it will be added to AWS CDK L2 constructs at some stage: https://github.com/aws/aws-cdk/issues/23153